zha/zigbee/cluster_handlers/homeautomation.py

```diff
@@ -116,7 +143,6 @@ class MeasurementType(enum.IntFlag):
         ElectricalMeasurement.AttributeDefs.measurement_type.name: True,
         ElectricalMeasurement.AttributeDefs.power_divisor.name: True,
         ElectricalMeasurement.AttributeDefs.power_multiplier.name: True,
-        ElectricalMeasurement.AttributeDefs.power_factor.name: True,
```

Comment:
Ok, I think this is almost good to go now, except for one thing:
All attributes that are exclusively in `ZCL_POLLING_ATTRS` like `power_factor` (and I guess also the `_max` attributes) need to be added to `ZCL_INIT_ATTRS` too, with `cache=True`, so they're initialized during pairing. Otherwise, the entities will not be created until ZHA is restarted.

(Attributes that are in `ATTR_REPORTING` should *not* be duplicated into `ZCL_INIT_ATTRS` though.)

There are some other PRs I want to get in first, so I'll look into slightly changing this behavior in the future, but to have less conflicts with the (IKEA) divisor/multiplier reporting PR, let's just add the three `power_factor` and I guess also the ten `_max` attributes to `ZCL_INIT_ATTRS`.